doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package enables you to visualize your `Iris cubes <http://scitools.org.uk/iris/docs/latest/userguide/iris_cubes.html>`_ in a Jupyter notebook.
+Cube Browser utilizes Python and the Matplotlib, Iris and Cartopy libraries while offering the additional functionality of plotting an entire cube without having to slice it.
 
-Links to Notebooks
-------------------
+You can make use of the Cube Browser library to write a short notebook to display your Iris cube data.
+There are several advantages of using this approach, most notably the flexibility in the options you can apply to each plot and the relative ease with which you can achieve this.
+Additionally, if you know what plots you would like and how you would like them laid out, this is a very quick way of producing your plots and exploring your cube data.
+You can also apply the same notebook to many different data sets at different times or with different customizations.
 
-four_axes.link
\ No newline at end of file
+Making Your Plots
+-----------------
+
+**Prepare your cube**
+
+Before you make your plot, you first have to choose and load the Iris cubes that you would like to visualize.
+If you need to perform any data manipulation on the cube, such as aggregating, collapsing or cube maths, this should also be done prior to plotting.
+
+**Configure axes**
+
+Cube Browser uses matplotlib functionality, so you must set up your axes in the layout that you desire for your plots.
+The steps that are required here are that you define your map projection
```

Comment:
Too much 'that'! The steps (define map proj and number of axes/subplots) should be displayed in a list so that they are easier to read. You can then follow the list up with 'For example::'